admin/nodes/urls.py

```diff
@@ -3,5 +3,8 @@
 from . import views
 
 urlpatterns = [
-    url(r'^$', views.NodeFormView.as_view(), name='node'),
+    url(r'^$', views.NodeFormView.as_view(),
+        name='search'),
+    url(r'^id-(?P<guid>[a-z0-9]+)/$', views.NodeView.as_view(),
```

Comment:
It really isn't. Something that I have picked up, but it is across so many PRs at this point... I would rather change them all at once, along with a good restructuring of many urls.